doobie-hikari/src/main/scala/com/avast/sst/doobie/DoobieHikariConfig.scala

```diff
@@ -0,0 +1,18 @@
+package com.avast.sst.doobie
+
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration.FiniteDuration
+
+final case class DoobieHikariConfig(driver: String,
+                                    url: String,
+                                    username: String,
+                                    password: String,
+                                    autoCommit: Boolean = true,
+                                    connectionTimeout: FiniteDuration = FiniteDuration(30, TimeUnit.SECONDS),
+                                    idleTimeout: FiniteDuration = FiniteDuration(10, TimeUnit.MINUTES),
+                                    maxLifeTime: FiniteDuration = FiniteDuration(30, TimeUnit.MINUTES),
+                                    minimumIdle: Int = 10,
+                                    maximumPoolSize: Int = 10,
+                                    readOnly: Boolean = false,
+                                    leakDetectionThreshold: Option[FiniteDuration] = None)
```

Comment:
or shouldn't this be
```scala
leakDetectionThreshold: FiniteDuration = FiniteDuration.Zero // or however you describe 0
```
:thinking: :thinking: :thinking:

Reply:
It could be but I like `Option` more, that's why I am converting it later.